mobile/android/android-log-filter.ts

```diff
@@ -29,15 +29,20 @@ export class AndroidLogFilter implements Mobile.IPlatformLogFilter {
 		return data + os.EOL;
 	}
 
-	private getConsoleLogFromLine(lineText: string): any {
-		let acceptedTags = ["chromium", "Web Console", "JS"];
+	private getConsoleLogFromLine(lineText: string, pid: string): any {
+		// filter log line if it does not belong to the current application process id
+		if (lineText.indexOf(pid) < 0) {
+			return null;
+		}
+
+		let acceptedTags = ["chromium", "Web Console", "JS", "ActivityManager", "System.err"];
 		let match = lineText.match(AndroidLogFilter.LINE_REGEX) || lineText.match(AndroidLogFilter.API_LEVEL_23_LINE_REGEX);
 
 		if (match && acceptedTags.indexOf(match[1].trim()) !== -1) {
 			return { tag: match[1].trim(), message: match[2] };
 		}
-		let matchingTag = _.some(acceptedTags, (tag: string) => { return lineText.indexOf(tag) !== -1; });
```

Comment:
Why is this line deleted? Its idea is to save us in cases where the logs are changed and our regex fails, but we still check if the line contains "chromium" for example.